plugin/pkg/scheduler/util/testutil.go

```diff
@@ -0,0 +1,168 @@
+/*
+Copyright 2017 The Kubernetes Authors.
```

Comment:
Some comments for this file:

1) If want to put the util here, then we should name it as `test_utils.go` to keep consistent.

2) But seems putting this to the folder of `testing` is better, we can just name the file as `testapi.go`.